### PR TITLE
add metrics on rack viz

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #2926 

image

### CODE REVIEW

File: inc/item_rack.class.php [hunk 1]
```diff
@@ -306,15 +305,18 @@ static function showItems(Rack $rack) {
...
+      echo '<div class="sep"></div>';
       echo "<div id='grid-dialog'></div>";
-      echo "</div>";
+      echo "</div>"; // #viewgraph
 
       $rack_add_tip = __s('Insert an item here');
       $ajax_url     = $CFG_GLPI['root_doc']."/ajax/rack.php";
 
-      $js = <<<JAVASCRIPT
+      $js = <<<HD
```
Comment (trasher) [inline inc/item_rack.class.php]: Why?

File: inc/item_rack.class.php [hunk 2]
```diff
@@ -499,10 +501,97 @@ classes: 'qtip-shadow qtip-bootstrap rack_tipcontent'
             }
          });
       });
-JAVASCRIPT;
+HD;
       echo Html::scriptBlock($js);
    }
 
+   static function showStats(Rack $rack) {
```
Comment (trasher) [inline inc/item_rack.class.php]: Missing phpdoc ;)

File: inc/item_rack.class.php (hunk 1 above)
Comment (orthagh) [reply inc/item_rack.class.php]: because :D

Comment (orthagh) [reply inc/item_rack.class.php]: i'll revert this part

**Pull request merged**